api/src/main/java/org/commonjava/indy/model/galley/CacheOnlyLocation.java

```diff
@@ -68,7 +68,7 @@ public boolean allowsPublishing()
     @Override
     public boolean allowsStoring()
     {
-        return repo != null && !repo.isReadonly();
+        return true;
     }
```

Comment:
Just do as the NOS-946, which mentioned the readonly will affect allowsDeletion, not allowsStoring.